plugin.yaml

```diff
@@ -150,6 +150,46 @@ node_types:
       cloudify.interfaces.validation:
         creation: aws.ec2.keypair.creation_validation
 
+  cloudify.aws.nodes.ElasticLoadBalancer:
+    derived_from: cloudify.nodes.LoadBalancer
+    properties:
+      elb_name:
+        type: string
+        description: >
+          name of load balancer
+      zones:
+        type: string
+        description: >
+          aws zones that the load balancer would operate in
+      security_groups:
```

Comment:
add default empty list
add in description: security groups only supported with vpc